docs/architecture/microservices/microservice-ddd-cqrs-patterns/seedwork-domain-model-base-classes-interfaces.md

```diff
@@ -9,7 +9,7 @@ The solution folder contains a *SeedWork* folder. This folder contains custom ba
 
 Figure 7-12 shows the classes that form the seedwork of the domain model in the ordering microservice. It has a few custom base classes like Entity, ValueObject, and Enumeration, plus a few interfaces. These interfaces (IRepository and IUnitOfWork) inform the infrastructure layer about what needs to be implemented. Those interfaces are also used through Dependency Injection from the application layer.
 
-![The detailed contents of the SeedWork folder, containing base classes and interfaces: Entity.cs, Enumeration.cs, IAggregateRoot.cs, IRepository.cs, IUnitOfWork.cs, and ValueObject.cs](./media/image13.PNG)
+![Screenshot of the classes contained in the SeedWork folder.](./media/seedwork-domain-model-base-classes-interfaces/vs-solution-seedwork-classes.png)
```

Comment:
We also need here the extended image tag, because the image is mostly text that makes no sense to include as content.

```markdown
:::image type="complex" source="<folderPath>" alt-text="<alt-text>":::
<long description here>
:::image-end:::
```